src/crypto/equihash.cpp

```diff
@@ -511,7 +511,7 @@ bool Equihash<N,K>::OptimisedSolve(const eh_HashState& base_state,
 
     const eh_index soln_size { 1 << K };
     std::vector<std::shared_ptr<eh_trunc>> partialSolns;
-    int invalidCount = 0;
+    size_t invalidCount = 0;
```

Comment:
Check L717 & L719. When printing either it should be `int invalidCount = 0;` or `printf("... %zu", invlidCount)`.